tests/network/conftest.py

```diff
@@ -230,3 +220,21 @@ def cnao_deployment(hco_namespace):
 @pytest.fixture(scope="session")
 def ovn_kubernetes_cluster(admin_client):
     return get_cluster_cni_type(admin_client=admin_client) == "OVNKubernetes"
+
+
+@pytest.fixture(scope="session", autouse=True)
+def network_sanity(hosts_common_available_ports, junitxml_plugin):
```

Comment:
@myakove For some reason, I don't see your comment, even though I see it it email notification. `@special_infra` is for fully capable BM clusters. PSI currently supports multi-nic too. Network team choose to not have their multi-nic tests marked, since their test jobs always have the requirement to run against a multi-nic cluster. Keeping the name to generic, because it can be expanded later.